modules/webgl/src/context/context.js

```diff
@@ -198,3 +194,13 @@ function logInfo(gl) {
   const debug = gl.debug ? ' debug' : '';
   log.once(1, `${webGL}${debug} context ${driver}`)();
 }
+
+function getVersion(gl) {
+  if (typeof WebGL2RenderingContext !== 'undefined' && gl instanceof WebGL2RenderingContext) {
+    return 2;
+  }
+  if (typeof WebGLRenderingContext !== 'undefined' && gl instanceof WebGLRenderingContext) {
+    return 1;
+  }
+  return null; // Not a WebGL context
```

Comment:
Nit: I think it's safe to assume you're getting a WebGL context here. I.e. just
```js
} else {
    return 1;
}
```

Reply:
Done.